bigbluebutton-html5/imports/ui/components/actions-bar/presentation-options/component.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { defineMessages, injectIntl, intlShape } from 'react-intl';
+import browser from 'browser-detect';
+import Button from '/imports/ui/components/button/component';
+import logger from '/imports/startup/client/logger';
+import { notify } from '/imports/ui/services/notification';
+import cx from 'classnames';
+import { styles } from '../styles';
+
+const propTypes = {
+  intl: intlShape.isRequired,
+  toggleSwapLayout: PropTypes.func.isRequired,
+};
+
+const intlMessages = defineMessages({
+  restorePresentationLabel: {
+    id: 'app.actionsBar.actionsDropdown.restorePresentationLabel',
+    description: 'Restore Presentation option label',
+  },
+  restorePresentationDesc: {
+    id: 'app.actionsBar.actionsDropdown.restorePresentationDesc',
+    description: 'button to restore presentation after it has been closed',
+  },
+});
+
+const BROWSER_RESULTS = browser();
+const isMobileBrowser = (BROWSER_RESULTS ? BROWSER_RESULTS.mobile : false) ||
```

Comment:
I think this may be remaining code from an older version of the HTML5 that I brought along when merging from our older branch into 2.2 ... I did not write this code and will remove it.